### PR TITLE
Sync with Operatorhub

### PR DESCRIPTION
**Pull request by slaskawi**
## Additional Information

This is a sync commit with the Operatorhub. Please merge this one after two PRs against the Operatorhub get in:

* https://github.com/operator-framework/community-operators/pull/1247
* https://github.com/operator-framework/community-operators/pull/1246

## Checklist:
- [ ] Verified by team member
- [ ] Comments where necessary
- [ ] Automated Tests
- [ ] Documentation changes if necessary

### CODE REVIEW

**Comment by slaskawi** [comment]
As the two PRs against Operatorhub were merged, @stianst could you please merge this one?

File: deploy/olm-catalog/keycloak-operator/8.0.2/keycloak-operator.8.0.2.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+    - Access
+  links:
+    - name: Documentation
+      url: https://github.com/keycloak/keycloak-operator/tree/8.0.2/docs
+    - name: Keycloak
+      url: https://www.keycloak.org/
+    - name: Keycloak Discourse
+      url: https://keycloak.discourse.group/
+  maintainers:
+    - email: keycloak-dev@lists.jboss.org
```
Comment (davidffrench) [inline deploy/olm-catalog/keycloak-operator/8.0.2/keycloak-operator.8.0.2.clusterserviceversion.yaml]: @slaskawi is this mailing list still correct?

Comment (slaskawi) [reply deploy/olm-catalog/keycloak-operator/8.0.2/keycloak-operator.8.0.2.clusterserviceversion.yaml]: That's a good catch!

I tried to update it to Google Groups in `8.0.1` release but it turns out they require an email address here (and there's no option to switch this to an arbitrary URL). I'll look if anything changed in `9.0.0` release.

For the time being, I would leave it as is - this is just a sync commit that pushes changes merged into Operatorhub.

Comment (davidffrench) [reply deploy/olm-catalog/keycloak-operator/8.0.2/keycloak-operator.8.0.2.clusterserviceversion.yaml]: ok, perfect.

**Comment by slaskawi** [comment]
@stianst May I ask you to merge this one?

**Pull request merged**